lib/util.js

```diff
@@ -489,3 +488,11 @@ api.callbackify = fn => {
     return fn.apply(null, args);
   };
 };
+
+// a replacement for jsonld.hasValue
+api.hasValue = (obj, key, value) => {
+  if(Array.isArray(obj[key])) {
```

Comment:
Why not add `const v = obj[key];` at the top to avoid look ups on `obj[key]` three times?

Reply:
did that in the loop bench, copied over here as well now, no perceptible performance change, but code is simpler I think.